statsmodels/kernel_methods/bandwidths.py

```diff
@@ -256,7 +264,7 @@ def getter(self):
         value = getattr(self._kde, attr)
         try:
             return getattr(self._kde, attr)[self._axis]
-        except:
+        except AttributeError:
```

Comment:
unit tests fail now because a `TypeError:` is raised that is not caught anymore
for example
https://travis-ci.org/statsmodels/statsmodels/jobs/438874375#L4434
with many similar errors